stl/inc/xhash

```diff
@@ -269,6 +269,12 @@ struct _Hash_vec {
             _Aliter_traits::max_size(_Mypair._Get_first()));
     }
 
+    _NODISCARD size_type capacity() const noexcept {
+        // This implementation never has capacity() differ from size(), but the previous implementation could.
+        // We need to handle that situation gracefully since we may link to old code.

```

Comment:
I recommend citing GH-2774 to provide additional detail. (The whole class is already TRANSITION, ABI so no need for that here.)